simulation/simulation_endpoint.py

```diff
@@ -11,3 +13,16 @@ class SimulationEndpoint(AutoMockEndpoint):
     async def open(self):
         self._open = True
         return succeed(None)
+
+    def get_link_latency(self, to_address):
+        """
+        Return the link latency to a particular destination address, in seconds.
+        """
+        return 0
```

Comment:
This is dead code. Please consider adding something like `self.latencies = defaultdict(int)` and changing this method to return `self.latencies[to_address]`.